content/en-us/glossary/vocabulary.md

```diff
@@ -19,6 +19,9 @@ Words and definitions used across the IoT Atlas.
 ### Endpoint
 > an endpoint is another name for a device, but often describing a device as an endpoint also implies the device operates on-premises with limited connectivity to the cloud
 
+### Industry 4.0 <a name="industry40"></a>
+> trend in manufacturing and automation to use IoT technology for connecting production assets to backend IT infrastructure (or cloud services) with the goal to extract data, gain finer grained control to production processes, allow predictions on quality and reduce downtimes ([Wikipedia]({{< ref "https://en.wikipedia.org/wiki/Industry_4.0" >}}))
```

Comment:
Causing render error due to use of `ref`.

Wikipedia link should be standard markdown like this: `([Wikipedia](https://en.wikipedia.org/wiki/Industry_4.0))`